### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by zhanwenzhuo-github**
### What problem does this PR solve?

Add corresponding issue link with summary if exists -->

Issue link:

### Code changes

- [ ] Has Code change
- [ ] Has CI related scripts change

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Note for reviewer

### CODE REVIEW

**Comment by zhanwenzhuo-github** [comment]
update issue template

**Comment by zhanwenzhuo-github** [comment]
add question.md

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: Question
+---
+
+**General Question**
+
+<!--
+
+Before asking a question, make sure you have:
+
+- Searched questions at https://github.com/infiniflow/infinity
```
Comment (JinHai-CN) [inline .github/ISSUE_TEMPLATE/question.md:13]: Replace the link with https://github.com/infiniflow/infinity/issues and https://github.com/infiniflow/infinity/discussions

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Googled your question 
+
+- Read the documentation at https://github.com/infiniflow/infinity
```
Comment (JinHai-CN) [inline .github/ISSUE_TEMPLATE/question.md:17]: Replace this link with https://github.com/infiniflow/infinity/tree/main/docs

**Comment by zhanwenzhuo-github** [comment]
question.md updated

**Pull request merged**